api/uploader/upload_asset.go

```diff
@@ -122,9 +122,11 @@ func (u *API) Upload(ctx context.Context, file interface{}, uploadParams UploadP
 	if err != nil {
 		return nil, err
 	}
+	//Parse playback_url from response
```

Comment:
This code is redundant, response parser automatically parses all relevant fields.

Reply:
> This code is redundant, response parser automatically parses all the relevant fields.

So do I need to remove this code (line 126,127)?

Reply:
@const-cloudinary

Reply:
yes, you can remove it.
You can also update [this test ](https://github.com/cloudinary/cloudinary-go/blob/74b7a5291c66ecc64545b170b5fbb7c197fc7a15/api/uploader/upload_asset_test.go#L93) and make sure that PlaybackURL is not empty

Reply:
@Suraj3240 ^